src/Components/TasksWrapper.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState, useEffect } from "react";
+import TaskList from "./TaskList";
+import TasksControls from "./TasksControls";
+import AddTaskModal from "./AddTaskModal";
+import Swal from "sweetalert2";
+
+export default function TasksWrapper() {
+  const [dataParsed, setDataParsed] = useState([
+    { id: 0, taskname: "Hello", assignee: "World", done: false },
+    { id: 1, taskname: "Hello2", assignee: "World2", done: true },
+  ]);
+  const [isData, setIsData] = useState(false);
+  const [toggle, setToggle] = useState(false);
+  const [token, setToken] = useState("");
+  const [tasksCount, setTasksCount] = useState(dataParsed.length);
+  const [completedTasksCount, setCompletedTasksCount] = useState(
+    dataParsed.filter((task) => task.done === true).length
+  );
+
+  function showSuccessPopup() {
+    Swal.fire("Good job!", "You clicked the button!", "success");
+  }
+
+  const parsingData = (data) => {
+    return JSON.stringify(data);
+  };
+
+  function getDataFromLocalStorage() {
+    const jsonFormatData = localStorage.getItem("Tasks");
+    console.log(jsonFormatData);
+    if (jsonFormatData) {
+      const todos = JSON.parse(jsonFormatData);
+      setDataParsed(todos);
+      setIsData(true);
+    }
+  }
+
+  function onAddClick(taskname, assignee) {
+    let dataObject = {
+      id: dataParsed.length,
+      done: false,
+      taskname,
+      assignee,
+    };
+
+    const updatedData = [...dataParsed, dataObject];
+    const localStorageData = parsingData(updatedData);
+
+    setDataParsed(updatedData);
```

Comment:
Did you notice that you are setting data and then syncing the local storage here and in lines 64 and 88?
This is an indicator that we need a function here.. 

```javascript
function syncLocalStorage() {
    const dataStringified = parseData(dataParsed);
    localStorage.setItem("Tasks", dataStringified);
}
```